pxtwinrt/hiddeploy.ts

```diff
@@ -73,13 +73,19 @@ namespace pxt.winrt {
                                         }
                                         this.onData(new Uint8Array(values));
                                     });
-                                } else {
-                                    pxt.debug(`no hid device found`);
+                                    return Promise.resolve();
                                 }
+                                pxt.debug("no hid device found");
+                                return Promise.reject(new Error("no hid device found"));
                             });
                     }
-                    else return Promise.resolve();
+                    return Promise.reject(new Error("no hid device found"));
```

Comment:
That's just an internal debug message that I reuse for an internal facing promise rejection error... we typically do not capitalize debug messages in the rest of the codebase